test/Tizen.NUI.ThemeManager.Samples/TizenThemeChangeSample/tizen-manifest.xml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="5.5" package="org.tizen.example.TizenThemeChangeSample" version="1.0.0">
+  <profile name="common" />
+  <ui-application appid="org.tizen.example.TizenThemeChangeSample"
+					exec="TizenThemeChangeSample.dll"
+					type="dotnet-nui"
+					multiple="false"
+					taskmanage="true"
+					nodisplay="false"
+					launch_mode="single"
+					api-version="6">
+    <label>TizenThemeChangeSample</label>
```

Comment:
```suggestion
    <label>TizenThemeChangeSample</label>
    <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true" />
```

Minor. Can you add AOT enable here?

Reply:
Great! Thank you for the suggestion. I added AOT key.